openstack/sharedfilesystems/v2/sharenetworks/testing/requests_test.go

```diff
@@ -168,3 +168,16 @@ func TestGet(t *testing.T) {
 
 	th.CheckDeepEquals(t, &expected, n)
 }
+
+// Verifies that it is possible to update a share network
+func TestUpdate(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+
+	MockUpdateResponse(t)
+
+	options := sharenetworks.UpdateOpts{Name: "net_my2"}
+	v, err := sharenetworks.Update(client.ServiceClient(), "713df749-aac0-4a54-af52-10f6c991e80c", options).Extract()
```

Comment:
This should be more thorough: all the `Update` fields should be tested. Also, the full `ShareNetwork` should be `Extract`ed and tested with `th.AssertDeepEquals`.